pvlib/modelchain.py

```diff
@@ -708,28 +799,40 @@ def ac_model(self, model):
     def infer_ac_model(self):
         """Infer AC power model from system attributes."""
         inverter_params = set(self.system.inverter_parameters.keys())
-        if {'C0', 'C1', 'C2'} <= inverter_params:
+        single_array = self.system.num_arrays == 1
+        if not single_array and {'C0', 'C1', 'C2'} <= inverter_params:
```

Comment:
I'd like a new `ValueError` that explains that a system with multiple arrays must include an inverter with keys for sandia_multi_inverter.

I hit this by first testing ModelChain with a system with a single array and `inverter_parameters = {'pdc0': 10}`. Inference worked as expected. Then I used a system with a second array and the same inverter parameters. The inference failed, but with a message that I think would be confusing to users.

I'd also like to see support for multiple arrays with the pvwatts inverter model. I don't think we're taking too many liberties with the model if we sum the DC power from each array and pass the result into `pvwatts_ac`. I'd perform the sum in `ModelChain.pvwatts_inverter`.

Reply:
> I'd also like to see support for multiple arrays with the pvwatts inverter model. I don't think we're taking too many liberties with the model if we sum the DC power from each array and pass the result into `pvwatts_ac`. I'd perform the sum in `ModelChain.pvwatts_inverter`.

I agree with this. Summing the DC power from each array is a logical way to extend the PVWatts model to inverters with multiple inputs.

Reply:
I'm not opposed to adding the sum to `inverter.pvwatts_ac` but that's a bit more aggressive than saying this is just a `ModelChain` feature. It would also run into some of the same type checking issues that complicated #1085 until we required the `sandia_multi_inverter` inputs to be tuples/lists.

Reply:
I'll make a PR to add `pvlib.inverter.pvwatts_multi` I think that a separate function helps distinguish the extension from the documented PVWatts model function.

Reply:
I separated the inference for singe- vs multi-array systems which allows for better error messages.